.travis.yml

```diff
@@ -4,6 +4,7 @@ python: 3.7
 before_install:
   - pip install pytest pytest-cov
   - pip install coveralls
+  - pip install pandas
```

Comment:
For now you could let `pandas` and `pytest` to be together installed by `pip install .[test]`.
you could:
```
pip install .[test]
pip install pytest-cov coveralls
```

to put them together.